lib/template/config/initializers/timeout.rb

```diff
@@ -0,0 +1,3 @@
+if Config.timeout > 0
```

Comment:
Good point, I was debating whether all the timeouts should be in the same file but makes more sense to have one per component/dependency instead.